llama_stack/distribution/build_container.sh

```diff
@@ -166,10 +172,10 @@ EOF
   # and update the configuration to reference the new container path
   python_cmd=$(get_python_cmd)
   external_providers_dir=$($python_cmd -c "import yaml; config = yaml.safe_load(open('$run_config')); print(config.get('external_providers_dir') or '')")
-  if [ -n "$external_providers_dir" ]; then
+  if [ -n "$external_providers_dir" ] && [ -d "$external_providers_dir" ]; then
     echo "Copying external providers directory: $external_providers_dir"
     add_to_container << EOF
-COPY $external_providers_dir /app/providers.d
+COPY $external_providers_dir /.llama/providers.d
```

Comment:
why do we have to change this? Isn't it breaking the lines below?

Reply:
@leseb I changed all instances of /app to /.llama lmk if that makes sense or if I am missing something